pkg/kubectl/resource/visitor.go

```diff
@@ -44,7 +42,7 @@ type Visitor interface {
 }
 
 // VisitorFunc implements the Visitor interface for a matching function
-type VisitorFunc func(*Info) error
+type VisitorFunc func(*Info, error) error
```

Comment:
@feihujiang, please add the godoc as @deads2k suggested. I think the HandleError() func would do the same job and looks cleaner, but I can live with this PR's approach.